com.neoutils.NeoRegex.yml

```diff
@@ -0,0 +1,33 @@
+app-id: com.neoutils.NeoRegex
```

Comment:
can you set up a link to https://github.com/NeoUtils/NeoRegex/ on https://www.neoutils.com/ ?

Reply:
I can yes

Reply:
Done